src/main/java/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -33,9 +34,22 @@ public BookDto findById(Long id) {
     }
 
     @Override
-    public List<BookDto> findAll() {
-        return bookRepository.findAll().stream()
-                .map(bookMapper::toDto)
-                .toList();
+    public List<Book> findAll() {
+        return bookRepository.findAll();
```

Comment:
if you are implement the mapping logic in the service, be consistent and use it here.